packages/lit-html/src/private-ssr-support.ts

```diff
@@ -57,6 +57,14 @@ export const _$LH = {
         return resolveOverrideFn(this, values);
       }
     },
+  patchDirectiveResolve: (
+    directiveClass: new (part: PartInfo) => Directive,
+    resolveOverrideFn: Directive['_$resolve']
```

Comment:
`_$resolve` is marked `/** @internal */` so won't be in the declaration, making it break when used on exported things. Unfortunately I think we need to revert the type simplification back to the explicit one.

Reply:
Good catch.